internal/provider/private_endpoint_services_resource.go

```diff
@@ -76,8 +76,22 @@ var endpointServicesSchema = schema.Schema{
 						Computed:    true,
 						Description: "Operation status of the service.",
 					},
+					"name": schema.StringAttribute{
+						Computed:    true,
+						Description: "Name of the endpoint service.",
+					},
+					"endpoint_service_id": schema.StringAttribute{
+						Computed:    true,
+						Description: "Server side ID of the private endpoint connection.",
+					},
+					"availability_zone_ids": schema.ListAttribute{
+						Computed:            true,
+						ElementType:         types.StringType,
+						MarkdownDescription: "AZ IDs users should create their VPCs in to minimize their cost.",
```

Comment:
This feels a little terse for something as eye-catching as "minimize their cost".   Maybe something like:
Availability Zone IDs of the private endpoint connection.  It's recommended to use these IDs for VPC placement for cost optimization.  See more here (some kind of docs link).

Reply:
done